app/src/main/res/navigation/main_nav.xml

```diff
@@ -176,7 +176,7 @@
 
         <argument
             android:name="decodedData"
-            app:argType="java.io.Serializable" />
```

Comment:
Does that mean we cannot use Serializable arguments in general? Or is it a problem in this specific case only, and if so, why?